src/edit/handler/Edit.SimpleShape.js

```diff
@@ -159,6 +176,8 @@ L.Edit.SimpleShape = L.Handler.extend({
 
 		if (marker === this._moveMarker) {
 			this._move(latlng);
+		} else if (marker === this._rotateMarker) {
+			this._move(latlng);
```

Comment:
Is this and the line under `_onTouchMove` going to be swapped with rotation-specific behavior, or is this already the desired behavior?

Reply:
I think this is the desired / default behavior. I'm not exactly sure how it works but I think much of this is "defaults" that get overridden in angular2